website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -445,28 +445,38 @@ spec:
           "The source of the intention. For a `type` of `consul` this is the name of a Consul service. The service doesn't need to be registered.",
       },
     },
+    {
+      name: 'Peer',
+      type: 'string: ""',
+      description: {
+        hcl:
+          "Specifies the [peer](/docs/connect/cluster-peering/index.mdx) of the source service. `Peer` is mutually exclusive with `Partition`.",
+        yaml:
+          "Specifies the [peer](/docs/connect/cluster-peering/index.mdx) of the source service. `peer` is mutually exclusive with `partition`.",
+      },
+    },
     {
       name: 'Namespace',
       enterprise: true,
-      type: 'string',
+      type: 'string: ""',
       description: {
         hcl:
-          "The namespace of the source service. Defaults to the namespace of the destination service (i.e. the config entry's namespace)",
+          "The namespace of the source service. If `Peer` is empty, `Namespace` defaults to the namespace of the destination service (i.e. the config entry's namespace).",
         yaml:
-          'The namespace of the source service. Defaults to the namespace of the destination service (i.e. `spec.destination.namespace`)',
+          'The namespace of the source service. If `peer` is empty, `namespace` defaults to the namespace of the destination service (i.e. `spec.destination.namespace`).',
       },
     },
     {
       name: 'Partition',
       enterprise: true,
-      type: 'string',
+      type: 'string: ""',
       description: {
         hcl:
-          "Specifies the admin partition of the source service. Defaults to the destination service's partition, i.e., the configuration entry's partition",
+          "Specifies the admin partition of the source service. If `Peer` is empty, `Partition` defaults to the destination service's partition (i.e. the configuration entry's partition). `Partition` is mutually exclusive with `Peer`.",
         yaml:
-          "Specifies the admin partition of the source service. Defaults to the destination service's partition, i.e. `spec.destination.partition`",
+          "Specifies the admin partition of the source service. If `peer` is empty, `partition` defaults to the destination service's partition (i.e. `spec.destination.partition`). `partition` is mutually exclusive with `peer`.",
```

Comment:
i noticed one is `Partition` while the other is `partition`. is this intentional?

Reply:
Yeah the YAML versions of all the docs use lowercase field names so I tried to be consistent